script.js

```diff
@@ -1,69 +1,103 @@
+/* eslint-disable max-classes-per-file */
 const addBookBtn = document.querySelector('.add-book');
 const bookName = document.querySelector('#title');
 const bookAuthor = document.querySelector('#author');
 const form = document.querySelector('.add-book-form');
 
 let booklist = [];
```

Comment:
- [ ] According to the requirements all the objects/arrays should have been replaced with JS classes, you can use objects and arrays but it has to be inside the classes

Reply:
Hi @mikethreels, that array is just used for reading in the list from local storage. Can you please suggest how we can change it?